src/api/cvc4cpp.h

```diff
@@ -2868,12 +2868,16 @@ class CVC4_PUBLIC Solver
    */
   Term simplify(const Term& t);
 
+  Term expandDefinitions(Term t) const;
+
   /**
    * Assert a formula.
    * SMT-LIB: ( assert <term> )
    * @param term the formula to assert
+   * @param isUnsatCore determines whether to include this asserted formula in
+   * an unsat core (if one is later requested)
    */
-  void assertFormula(Term term) const;
+  void assertFormula(Term term, bool isUnsatCore = true) const;
```

Comment:
I believe we decided that this would be a more general name like `isTracked`.  You should then say that including this asserted formula in the unsat is an *example* of what is implied by setting this true.  I believe there were other examples of what this would impact (e.g. getAssignment).  @4tXJ7f suggested this?

Reply:
Discussed offline. Removed the option.